src/tests/Interop/COM/Activator/Activator.csproj

```diff
@@ -4,8 +4,8 @@
     <RequiresMockHostPolicy>true</RequiresMockHostPolicy>
     <!-- The test fails casting from ClassFromA from the default ALC to type IGetTypeFromC from a custom ALC -->
     <UnloadabilityIncompatible>true</UnloadabilityIncompatible>
-    <DisableProjectBuild Condition="'$(RuntimeFlavor)' == 'Mono'">true</DisableProjectBuild>
```

Comment:
Deleting this because the same line is already in src/tests/Interop/COM/Directory.Build.props.

Btw in the same props file we also disable building everything under /COM outside Windows, but it looks like there is at least one test under COM that is specifically for Unix (GlobalInstanceTrackerSupportTests_TargetUnix.csproj).